tokio-rustls/tests/client.cert

```diff
@@ -0,0 +1,27 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I don't know how these were generated? It would be good to make that explicit in the script, ideally as part of the `tokio-native-tls/scripts/generate-certificate.sh` script.